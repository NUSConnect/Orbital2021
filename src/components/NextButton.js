import React from 'react'
import { Text, TouchableOpacity } from 'react-native'

export default function NextButton ({ ...props }) {
  return (
    <TouchableOpacity style={{ marginHorizontal: 10 }} {...props}>
      <Text style={{ fontSize: 16 }}>Next</Text>
    </TouchableOpacity>
  )
}
