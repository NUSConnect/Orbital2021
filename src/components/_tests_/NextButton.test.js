import * as React from 'react'
import NextButton from '../NextButton'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(<NextButton />)
  expect(tree).toMatchSnapshot()
})
