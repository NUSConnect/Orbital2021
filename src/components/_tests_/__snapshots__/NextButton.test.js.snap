// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "marginHorizontal": 10,
      "opacity": 1,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 16,
      }
    }
  >
    Next
  </Text>
</View>
`;
